src/lib/auth.ts

```
@@ -0,0 +1,79 @@
+  //         },
+  //       },
+  //     },
+  //   }),
+  // ),
+
+  providers: [
+    ...authConfig.providers,
+    NodeMailer({
+      server: {
+        host: process.env.EMAIL_SERVER_HOST,
+        port: Number.parseInt(process.env.EMAIL_SERVER_PORT!),
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)